fdk_rdf_parser_service/model.py

```diff
@@ -0,0 +1,37 @@
+from typing import Literal, Union, Dict
+
+from fdk_rdf_parser.classes import (
+    Dataset,
+    Concept,
+    InformationModel,
+    PublicService,
+    Event,
+    DataService,
+)
+
+ResourceType = Union[
+    Dataset,
+    Concept,
+    InformationModel,
+    PublicService,
+    Event,
+    DataService,
+]
+
+CatalogType = Literal[
+    "datasets",
+    "data-services",
+    "concepts",
+    "information-models",
+    "services",
+    "events",
+]
+
+catalog_type_map: Dict[str, CatalogType] = {
```

Comment:
hva er vitsen med denne?
key og value er jo alltid like...

Reply:
For å ha typesafety på statisk typesjekk, samtidig som definisjonen av endepunktet kan gjøres dynamisk. Fant ingen måte å "materialisere" Literals til faktisk streng-instanser, tror ikke det er mulig i Python.

Reply:
har skrevet om til enum